src/core/server/saved_objects/migrations/core/document_migrator.test.ts

```diff
@@ -748,10 +748,7 @@ describe('DocumentMigrator', () => {
         migrator.migrate(_.cloneDeep(failedDoc));
         expect('Did not throw').toEqual('But it should have!');
       } catch (error) {
-        expect(error.message).toMatchInlineSnapshot(`
-          "Failed to transform document smelly. Transform: dog:1.2.3
-          Doc: {\\"id\\":\\"smelly\\",\\"type\\":\\"dog\\",\\"attributes\\":{},\\"migrationVersion\\":{}}"
-        `);
+        expect(error.message).toMatchInlineSnapshot(`"Dang diggity!"`);
```

Comment:
So, we don't know during which SO the migration the function throws an exception?

Reply:
@mshustov We have the raw id in the error message that's returned in the logs when the migration fails. 

Do you think we need to [prefix](https://github.com/elastic/kibana/pull/100297/files#diff-f65f63dbdc98ed06155d38fa35e23643184ba81afe28f36131d0e2ef3a58e5e9R16) the `originalError.message` with the [SO uuid](https://github.com/elastic/kibana/pull/100297/files#diff-f65f63dbdc98ed06155d38fa35e23643184ba81afe28f36131d0e2ef3a58e5e9L23) we have in `document_migrator` or append it to the [message](https://github.com/elastic/kibana/blob/master/src/core/server/saved_objects/migrations/core/migrate_raw_docs.ts#L110-L116) before returning it from the `migrationRawDocSafely` or do something else?

Reply:
> we have the raw id in the error message that's returned in the logs when the migration fails.

Do we have any test asserting the error message contains SO.id?

>  thought the intention was to surface the original error's message and show that in the list of failures.

Right, as long as we can understand which SO document caused a problem.